addons/account/models/chart_template.py

```diff
@@ -183,7 +191,6 @@ def _load(self, template_code, company, install_demo):
             install_demo = False
         data = self._pre_load_data(template_code, company, template_data, data)
         self._load_data(data)
-        self._load_translations(companies=company)
```

Comment:
l moved it to the end of the function and did not find any problem with it. Is it a problem?